src/rand.rs

```diff
@@ -432,3 +435,34 @@ mod fuchsia {
         fn zx_cprng_draw(buffer: *mut u8, length: usize);
     }
 }
+
+#[cfg(any(target_env = "sgx"))]
+mod sgx {
+    use core::arch::x86_64::_rdrand64_step;
+    use crate::error;
+
+    #[inline]
```

Comment:
`#[inline]` does nothing on a private function, the attribute is only useful for `pub` functions/methods.